python/Ganga/GPIDev/Lib/File/MassStorageFile.py

```diff
@@ -405,14 +411,15 @@ def validate(self):
 
         return (True, '')
 
-    def handleUploadFailure(self, error):
+    def handleUploadFailure(self, error, cmd=''):
```

Comment:
You've named this argument `cmd` (implying it's the command that was run) but most of the time you;re passing it something like `'1) %s %s' % (ls_cmd, pathToDirName)` which is not a command.